common/namespace/transmissionTaskHandler_test.go

```diff
@@ -73,7 +73,7 @@ func (s *transmissionTaskSuite) TearDownTest() {
 
 func (s *transmissionTaskSuite) TestHandleTransmissionTask_RegisterNamespaceTask_IsGlobalNamespace() {
 	taskType := replicationgenpb.ReplicationTaskType_NamespaceTask
-	id := primitives.NewUUID()
+	id := primitives.NewUUID().String()
```

Comment:
Used regexes to do most of this, hence different methods of string creation of UUIDs. 

I plan to follow up with an additional change to remove direct references to google/pborman UUID so `uuid.New()` and `uuid.NewRandom()` will instead use our `primitives.UUID`.